cypress/e2e/token-details.test.ts

```diff
@@ -144,13 +144,14 @@ describe('Token details', () => {
       cy.get('#swap-currency-output .token-amount-input').clear().type('0.0').should('have.value', '0.0')
     })
 
-    it('should show a L2 token even if the user is connected to a different network', () => {
+    it.only('should show a L2 token even if the user is connected to a different network', () => {
```

Comment:
```suggestion
    it('should show a L2 token even if the user is connected to a different network', () => {
```

Reply:
@JFrankfurt for some reason there's another test in this file using `it.only`, so i need this for it to be included

Reply:
do the others run if you remove the .only on the other test?